pyproject.toml

```
@@ -54,7 +54,7 @@ benchmark = [
 
 
 [project.entry-points.sklearn_engines]
-kmeans = "sklearn_numba_dpex.cluster.kmeans:KMeansEngine"
+kmeans = "sklearn_numba_dpex.kmeans.drivers:KMeansSklearnEngine"
```

Maybe this would feel more natural to me (avoid putting sklearn everywhere).

```suggestion
kmeans = "sklearn_numba_dpex.kmeans.engine:KMeansEngine"
```